docker/vue-app/src/components/dialog_content/ImportFormDialog.js

```diff
@@ -0,0 +1,59 @@
+export default {
+    data() {
+        return {
+            initialFocusElID: 'formPacket',
+            files: null,
+        }
+    },
+    inject: [
+        'APIroot',
+        'CSRFToken',
+        'closeFormDialog',
+        'selectNewCategory'
+    ],
+    mounted() {
+        document.getElementById(this.initialFocusElID).focus();
+    },
+    methods: {
+        onSave() {
+            if (this.files !== null) {
+                let pkg = new FormData();
+                pkg.append('formPacket', this.files[0]);
+                pkg.append('CSRFToken', this.CSRFToken);
+                
+                $.ajax({
+                    type: 'POST',
+                    url: `${this.APIroot}formStack/import`,
+                    processData: false,
+                    contentType: false,
+                    cache: false,
+                    data: pkg,
+                    success: (res) => {

```

Comment:
I removed the log, but I agree there is room for improving user feedback here.  The current live behavior is to return the user to the form editor if the response was true, or echo the response otherwise.  For now, this is just consistent with current behavior.